audit-connector/src/main/java/com/sequenceiq/cloudbreak/audit/converter/ResultApiRequestResultBuilderUpdaterAttempt.java

```diff
@@ -0,0 +1,26 @@
+package com.sequenceiq.cloudbreak.audit.converter;
+
+import com.cloudera.thunderhead.service.audit.AuditProto;
+import com.sequenceiq.cloudbreak.audit.model.ResultApiRequestData;
+import org.apache.commons.lang3.StringUtils;
+import org.springframework.stereotype.Component;
+
+import static com.sequenceiq.cloudbreak.util.ConditionBasedEvaluatorUtil.doIfTrue;
+
+@Component
+public class ResultApiRequestResultBuilderUpdaterAttempt implements AttemptAuditEventResultBuilderUpdater<ResultApiRequestData> {
```

Comment:
`ResultApiRequestResultBuilderUpdaterAttempt` is a way overcomplicated name.
I would have still stuck to naming the interface to reflect the converter nature: `AttemptAuditEventResultConverter`
and then name this class like `ResultApiRequestConverter`
Thoughts?

Reply:
but it's still not converting anything :D
Anyway I've renamed the class to refer to the implemented generic class